### DISCUSSION TITLE
0.33.5 Beta Github Copilot Chat Configure

### DISCUSSION THREAD

**Discussion by burakCokyildirim** [intitni/CopilotForXcode#551] [General]
I can't figure out how to make [this](https://github.com/intitni/CopilotForXcode/releases/tag/0.33.5.beta) setting, can you help me?

> To use it, you have to turn on "Pretend IDE to be VSCode" in the GitHub Copilot settings.

**Comment by intitni** [comment]
<img width="553" alt="Screenshot 2024-07-02 at 21 15 58" src="https://github.com/intitni/CopilotForXcode/assets/793147/478cfe7c-215e-4a00-b4e4-d6ea0f37e6bd">

Don't forget to refresh the configuration after turning on the toggle.

**Reply by burakCokyildirim** [reply]
Thank you 🙏
